README.md

```diff
@@ -1,9 +1,18 @@
 # Godot Website
 
-This repository contains the theme and plugins used in Godot Engine's
-October instance.
+This repository contains the theme and plugins used by the Godot Engine's
```

Comment:
Looks like the repo no longer contains the theme

Reply:
Ah yes, fair enough